stable/kibana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kibana
-version: 3.0.0
+version: 3.1.0
```

Comment:
Is this a breaking change for people who used 3.0 before?

Reply:
No, I'll change it to `3.0.1`